job-dsls/src/main/resources/job-scripts/prod_osl_trigger_umb_message.jenkinsfile

```diff
@@ -53,10 +55,20 @@ pipeline {
                 }
             }
         }
+        stage('Fetch images tags') {
+            steps {
+                script {
+                    imagesTags["data-index-ephemeral"] = sh (
+                        script: "brew buildinfo ${env.DATA_INDEX_EPHEMERAL_BREW} | awk -F'Extra: ' '{print \$2}' | tr \\\' \\\" | sed 's|False|\"false\"|g' | sed 's|None|\"\"|g' | jq -e '.image.index.pull[0]'",
```

Comment:
I think we should add also the handling of `True` value :)
```suggestion
                        script: "brew buildinfo ${env.DATA_INDEX_EPHEMERAL_BREW} | awk -F'Extra: ' '{print \$2}' | tr \\\' \\\" | sed 's|False|\"false\"|g' | sed 's|True|\"true\"|g' | sed 's|None|\"\"|g' | jq -e '.image.index.pull[0]'",
```

Reply:
Thanks, added!